web_src/css/repo/home-file-list.css

```diff
@@ -68,8 +103,7 @@
   color: var(--color-text-light-1);
 }
 
-@media (max-width: 767.98px) {
-  #repo-files-table .repo-file-cell.name {
-    max-width: 150px;
-  }
+#repo-files-table .parent-link {
+  width: 100%;
+  text-decoration-line: none;
```

Comment:
Don't see why it needs to write this style again.

![image](https://github.com/user-attachments/assets/e66a26ba-f97a-4cbb-834f-ec8d812e9611)

Reply:
This is to remove the hover underline. Could change selector to `:hover` if your prefer.

Reply:
Well, I can see this:

![image](https://github.com/user-attachments/assets/2492e421-02d9-4ca7-bf92-58f9cf08742c)

Reply:
It's not quite the same style as silenced class does not color on hover, but let's use it anyways. I don't feel like introducing another link class.